fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -247,11 +255,10 @@ ComplianceMsg=""
 # -- Slice timing correction
 
 if [ "${DoSliceTimeCorrection}" = 'TRUE' ]; then
-  ComplianceMsg+=" --doslicetime=TRUE"
+  ComplianceMsg+=" --doslicetime=TRUE WARNING: This legacy option of slice timing correction is performed before motion correction (as is typically done in legacy-style brain imaging) and thus assumes that the brain is motionless. Errors in temporal interpolation will occur in the presence of head motion and may also disrupt data quality measures as shown in Power et al 2017 PLOS One 'Temporal interpolation alters motion in fMRI scans: Magnitudes and consequences for artifact detection.' Slice timing correction and motion correction would ideally be performed simultaneously; however, this is not currently supported by any major software tool. HCP-Style fast TR fMRI data acquisitions (TR<=1s) avoid the need for slice timing correction, provide major advantages for fMRI denoising, and are recommended."
```

Comment:
Ditto, the warning should be a separate log_Warn call, not added to the string.

Reply:
Ok. That makes the code much more complicated though.

Reply:
Implemented here and in the shlib.

Reply:
Per the comments on the mppcheck shlib, the log_Warn call should be here, not in the check function.

Reply:
Done.